lager/lenses/tuple.hpp

```diff
@@ -0,0 +1,194 @@
+#pragma once
+
+#include <tuple>
+#include <type_traits>
+
+#include <lager/lenses.hpp>
+#include <lager/lenses/attr.hpp>
```

Comment:
Nitpick: I prefer to include headers with more local to less local order. I.e. lager ones first, std ones later.